core/src/main/scala/io/finch/syntax/ToTwitterFuture.scala

```diff
@@ -0,0 +1,26 @@
+package io.finch.syntax
+
+import cats.Monad
+import com.twitter.util.Future
+import io.catbird.util._
+
+/**
+  * Type class for conversion of some HKT (i.e. scala.concurrent.Future) to twitter future
+  */
+trait ToTwitterFuture[F[_]] {
+
+  def M: Monad[F]
```

Comment:
Why is this necessary?

Reply:
To make generic testing via behaviour possible